content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md

```diff
@@ -33,28 +33,47 @@ Corda now supports JDK Azul 8u382 and Oracle JDK 8u381.
 
 ### Ledger Recovery
 
+Ledger Recovery was introduced as part of the Corda 4.11 release. It complements a standardised Corda network operational backup and recovery process.
+
+For more information, see [Ledger Recovery]({{< relref "node/collaborative-recovery/ledger-recovery/overview.md" >}}).
+
+#### Two Phase Finality
+
+Two Phase Finality protocol (`FinalityFlow` and `ReceiveFinalityFlow` sub-flows) has been added to improve resiliency and
+recoverability of CorDapps using finality. Existing CorDapps do not require any changes to take advantage of this
+new improved protocol.
+
+For more information, see [Two Phase Finality]({{< relref "two-phase-finality.md" >}}).
+
+#### Finality Recovery Tooling
+
+RPC extension operations (additions to the FlowRPCOps interface) which allow for Finality Flow recovery by both the initiator and the receiver(s).
+Also, Node Shell commands now allow operations teams to perform Finality Flow recovery.
+
+For more information, see [Finality Flow Recovery]({{< relref "finality-flow-recovery.md" >}})
+
+#### Ledger Recovery flow
+
 A new ledger recovery flow (`LedgerRecoveryFlow`) enables a node to identify and recover transactions from
 peer recovery nodes to which it was a party (either initiator or receiver) and which are missing from its own ledger.
 
-For more information, see [Ledger Recovery flow]({{< relref "node/collaborative-recovery/ledger-recovery/ledger-recovery-flow.md" >}}).
+For more information, see [Ledger Recovery flow parameters]({{< relref "node/collaborative-recovery/ledger-recovery/ledger-recovery-flow.md" >}}).
 
-### Confidential Identity key-pair generator
+#### Confidential Identity key-pair generator
 
 A new service has been added that pregenerates Confidential Identity keys to be used when using CIs in transactions.
 These pre-generated CIs are subsequently used for backup recovery purposes.
 
-### Additional Network Parameters
+#### Additional Network Parameters
 
 The following network parameters, and associated node configuration parameters, have been added:
 
 * `confidentialIdentityMinimumBackupInterval`
-* `confidentialIdentityPreGenerationPeriod`
 * `recoveryMaximumBackupInterval`
-* `transactionRecoveryPeriod`
 
 For more information, see [Available Network Parameters]({{< relref "network/available-network-parameters.md" >}}).
```

Comment:
Can we add something that say that setting these will require CENM 1.6 or later, please.